src/ess/polarization/correction.py

```diff
@@ -208,3 +234,27 @@ def PolarizationAnalysisWorkflow(
         TransmissionFunction[Analyzer]
     ]
     return workflow
+
+
+def HalfPolarizedAnalysisWorkflow(
```

Comment:
If this is not too big of a change, I recommend to call this `HalfPolarizationWorkflow`. (As long as it is defined clearly this name does not really matter, but most people understand the "FullPolarization" - with Analyzer - when "Analysis" is mentioned.)